docs/globals.mdx

```diff
@@ -6,28 +6,20 @@ Sometimes you might want to insert global css like resets or font faces. You can
 
 ```jsx
 // @live
-import { Global, css } from '@emotion/core'
+import { Global } from '@emotion/core'
 
 render(
   <div>
-    <Global
-      styles={css`
-        * {
-          color: hotpink !important;
-        }
-      `}
-    />
     <Global
       styles={{
         '.some-class': {
+          color: 'hotpink !important',
```

Comment:
Could you move this back to the string one but make it target `.some-class`(I want there to still be a string example)